frontend/src/views/Home.js

```diff
@@ -4,26 +4,31 @@ import ViewWrappers from "../components/ViewWrappers/ViewWrappers";
 import Map from "../components/Map/Map";
 
 import './Home.scss';
+import useHomeLocationQuery from 'context/useHomeLocationQuery';
 
-const Home = () => (
-  <ViewWrappers.View>
-    <ViewWrappers.ViewSplitter small>
-      <div className="Home">
-        <div className="Intro">
-          <h1>Support your favourite local store online.</h1>
-          <p>Covexit enables small businesses to create an easy-to-use online shop to be able to master the corona crisis.</p>
-        </div>
+const Home = () => {
+  const [handleGeoLocation] = useHomeLocationQuery();
 
-        <div className="Btn-group">
-          <Button to="/stores" label="Search for shops nearby!" />
-          <Button to="/stores/new" label="Register your business" secondary />
+  return (
+    <ViewWrappers.View>
+      <ViewWrappers.ViewSplitter small>
+        <div className="Home">
+          <div className="Intro">
+            <h1>Support your favourite local store online.</h1>
+            <p>Covexit enables small businesses to create an easy-to-use online shop to be able to master the corona crisis.</p>
+          </div>
+  
+          <div className="Btn-group">
+            <Button onClick={handleGeoLocation} to="/stores" label="Search for shops nearby!" />
```

Comment:
and where do you pass the location to the map?

Reply:
Oh!, the map component isn't expecting location prop.

Reply:
Do I need to add more marker with the user location

Reply:
No no no, this task is so that the map centers on the users current position

Reply:
Oh Okay.